src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
+type Color = 'red' | 'green' | 'blue';
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+
 export interface Figure {
+  shape: Shapes;
+  color: Color;
+  getArea(): number;
+}
 
+function isBadTriangle(
```

Comment:
Maybe you can make this function as a method of Circle. It looks like it is a part of class. But it's on you own decision.